src/beautifiers/eslint.coffee

```diff
@@ -14,6 +14,7 @@ module.exports = class ESLintFixer extends Beautifier
   }
 
   beautify: (text, language, options) ->
+    context = this
```

Comment:
I do not think this self referential is necessary.
See https://stackoverflow.com/a/8965984/2578205 and https://coffeescript.org/#fat-arrow

Alternatively:

Before:

```coffeescript
context = this
# .... 
context.which('eslint').then(...)
```

After:

```coffeescript
eslint = this.which('eslint')
# .... 
eslint.then(...)
```

Also note `this` should be replaced with `@` in CoffeeScript. See https://coffeescript.org/#operators

Reply:
Yeah, it is better.
I will change the code and update my pull request.